src/schema/objects/columns.yaml

```diff
@@ -42,6 +42,31 @@ age:
     for privacy purposes.
   type: number
   unit: year
+age_category:
+  name: age_category
+  display_name: Age category
+  description: |
+    For non-numeric description of the age (e.g. developmental stage) a string can be provided.
+    We RECOMMEND to use the [UBERON life cycle stage](http://purl.obolibrary.org/obo/UBERON_0000105) to specify it.
+  type: string
+age_type:
+  name: age_type
+  display_name: Age type
+  description: |
+    Type of age measurement (e.g. `postnatal`, `fetal`, `gestational`). If not specified, `postnatal` is assumed.
```

Comment:
is there some ontology or controlled vocab to reference here as well by any chance?

Reply:
if `age_reference`, some time points to consider:

- fertilization
- birth
- gestation
- ...

it seems we are boiling down to establishing a list of Enums with clear definitions and possibly adding TermURLs for some of them where there is a clear association.  

TODO also to formalize such "TermURL" like linking at the level of schema which seems to lack it ATM.